aten/src/ATen/native/cuda/Copy.cu

```diff
@@ -355,19 +359,17 @@ static void copy_kernel_cuda(TensorIterator& iter, bool non_blocking) {
     auto conversion_device = non_blocking ? kCUDA : kCPU;
     if (iter.device_type(1) == conversion_device) {
       dst_contig = dst.is_contiguous() ? dst : at::empty_like(dst, LEGACY_CONTIGUOUS_MEMORY_FORMAT);
-      src_contig = iter.tensor(1).to(iter.dtype(0)).expand_as(dst).contiguous();
+      src_contig = iter.tensor(1).to(iter.dtype(0)).expand_as(dst).contiguous().resolve_conj();
```

Comment:
what if src `is_conj` is false and `dst` is_conj is true? `resolve_conj` will do nothing, `dst_contig.copy_(src_contig)` will still require temporaries, you are in an infinite loop.